samples/subsys/zbus/msg_subscriber/prj.conf

```diff
@@ -2,9 +2,8 @@ CONFIG_LOG=y
 CONFIG_LOG_MODE_MINIMAL=y
 CONFIG_ASSERT=y
 CONFIG_BOOT_BANNER=n
-CONFIG_MAIN_THREAD_PRIORITY=5
+CONFIG_MAIN_THREAD_PRIORITY=3
```

Comment:
I'm curious on the rationale behind bumping up the priority on this sample. Is it a related change?

Reply:
This was necessary to ensure the correct sequence of outputs and to cover the code with the sample working as a test.